backend/de.metas.business/src/main/java-legacy/org/compiere/model/MInvoice.java

```diff
@@ -1463,6 +1467,7 @@ public boolean reverseCorrectIt()
 		}
 		reversal.setC_Payment_ID(0);
 		reversal.setIsPaid(true);
+		reversal.setIsPartiallyPaid(false);
```

Comment:
@metas-ts  @dm-ps don't we need to set the openamt=0?

Reply:
it will be set from `de.metas.invoice.service.impl.AbstractInvoiceBL#testAllocation`